main.css

```diff
@@ -0,0 +1,575 @@
+    color: #ff5438;
+
+    /* font-size: 14.3px; */
+    font-weight: 700;
+    line-height: 25px;
+  }
+
+  .card-text {
+    height: 90%;
+    font-weight: normal;
+    width: 100%;
+  }
+
+  .join {
+    width: 75%;
+    height: 80px;
+    background-color: #ff4f38;
+    color: #fff;
+    margin-top: 30px;
+    font-weight: 900;
+    font-size: 20px;
+  }
+
+  .see-prog {
+    display: none;
+  }
+
+  .speakers-card {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .speakers {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .speaker {
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+    align-items: center;
+    margin-top: 20px;
+    margin-left: 30px;
+
+    /* border: #ec5242 dashed 2px; */
+  }
+
+  .speaker-img {
+    width: 106px;
+    height: 106px;
+  }
+
+  .wrapper {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    margin-left: 10px;
+  }
+
+  .speaker-name {
+    font-size: 15px;
+    font-weight: 700;
+  }
+
+  .speaker-title {
+    color: #f78c89;
+    font-size: 10px;
+    font-weight: 700;
+  }
+
+  .speaker-desc {
+    font-size: 9px;
+    line-height: 17px;
+    font-weight: 700;
+  }
+
+  .hide-mobile {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  header {
+    height: 100%;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    overflow: hidden;
+  }
+
+  .hidden {
+    display: none;
+  }
+
+  body::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    opacity: 0.1;
+    z-index: -1;
+    background-image: url('./img/back-img1.png');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+
+    /* background-color: #fff; */
+    padding-top: 50px;
+    background-clip: border-box;
+  }
+
+  .list-1 {
+    display: flex;
+    background-color: #000b;
+    justify-content: end;
+    color: white;
+    padding-right: 250px;
+    height: 25px;
+  }
+
+  ul {
+    list-style: none;
+  }
+
+  .list-1 ul {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  li a {
+    color: #fff;
+  }
+
+  .mobile-menu {
+    display: none;
+  }
+
+  .nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    gap: 40px;
+    background-color: #fff;
+    padding: 0 150px 0 150px;
+  }
+
+  .logo {
+    width: 70px;
+    height: 50px;
+  }
+
+  .nav ul {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+  }
+
+  .list-h4 {
+    color: #ec5242;
+    border: 4px solid #ec5242;
+    padding: 5px;
+  }
+
+  .nav a:hover {
+    color: #ec5242;
+  }
+
+  .description {
+    display: flex;
+    flex-direction: column;
+    width: 800px;
+    margin: 100px 200px;
+    text-align: left;
+  }
+
+  .desc-header {
+    font-size: 25px;
+    color: #f00;
```

Comment:
- Great Job for fixing the previous reviewer's suggestions,👌 I think you missed one thing from the previous reviewer which is updating colors based on the original design.
* kindly use thoase color swatch: #272A31, #EC5242, #D3D3D3 to stick the given design

![Uploading 111.JPG…](https://user-images.githubusercontent.com/61408860/234608705-047bb1f7-d7b4-4c13-baf8-d4f619edbec4.png)